src/aap_eda/services/activation/manager.py

```diff
@@ -102,11 +102,12 @@ def __init__(
             db_instance: The database instance of the activation.
             container_engine: The container engine to use.
         """
-        self.db_instance = db_instance
+        self.db_instance = proxy_instance.instance
+        self.proxy_instance = proxy_instance
         if container_engine:
             self.container_engine = container_engine
         else:
-            self.container_engine = new_container_engine(db_instance.id)
+            self.container_engine = new_container_engine(proxy_instance.id)
```

Comment:
@Alex-Izquierdo Will this id be unique across both the Activation and Source class. Otherwise the engine will create labels that might match multiple objects

Reply:
@mkanoor My first approach is to keep them separate to respect the requirement of treating them as completely different objects, which allows for the possibility of adding more types in the future or implementing the relationships between the child objects through generic relations. In this way, the manager needs to be updated to create the activation instance according to the type of the ProcessParent. This will be addressed in the next PR to implement the management and execution of sources.

I think implementing a unique id among sources and activations is possible in different ways, but at this point, I feel it's not a good idea. Anyway, I don't have enough arguments in favor or against, so we can discuss it in more detail if you want.

Reply:
@Alex-Izquierdo I am talking about stuff like this
https://github.com/ansible/eda-server/blob/01746bec37107c64bd0e11a58adb6600015e6951/src/aap_eda/services/activation/engine/kubernetes.py#L95
If the 2 ids are the same this code is going to have issues.

Reply:
Yes, I'm aware, we need to change this to deal with sources and activations, is part of the scope for the next PR. We were using the string "activation" to name our resources, now we will have to use activation or source. 
I thought the container request might contain this information.

Reply:
I have not looked at that in deep but I think we are passing the activation_id to the engine only for this type of thing. I want to look if we really need to receive the id there.